### PR TITLE
Issue53 libbpf and deploy script update

### PR DESCRIPTION
**Pull request by kiran1048**
Following files/submodules updated:
.travis.yml
README.md
deploy/build/zeta_dev.Dockerfile
deploy/deploy_common.sh
docs/user/getting_started.md
src/extern/alcor-control-agent
src/extern/libbpf
src/xdp/CMakeLists.txt

### CODE REVIEW

File: deploy/deploy_common.sh [hunk 1]
```diff
@@ -21,11 +21,6 @@ myloc=$(pwd)
 # Get full path of this script no matter where it's placed and invoked
 MY_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-if [[ ! -f $MY_PATH/playbooks/inventories/vars/.vault.yml ]]; then
```
Comment (liangbin-pub) [inline deploy/deploy_common.sh:24]: Don't make such change, env_setup is needed for local setup, you can touch create the needed file or add new condition to filter out situation.

File: deploy/deploy_common.sh [hunk 2]
```diff
@@ -45,7 +40,7 @@ if [ "$#" -eq 0 ] || [ "$1" == "" ] ; then
     echo
     read -p " please enter your preference: [d|p|u|r]: " opt
     case $opt in
-      [d]* )  
+      [d]* )
```
Comment (liangbin-pub) [inline deploy/deploy_common.sh:43]: Don't make whitespace only changes

File: deploy/deploy_common.sh [hunk 3]
```diff
@@ -100,13 +95,14 @@ else
...
 
-  if ! $dpur_opt 
+  if ! $dpur_opt
   then
     echo " must enter operation!" >&2
     exit 1
   fi
 fi
 
+
```
Comment (liangbin-pub) [inline deploy/deploy_common.sh:105]: Don't make whitespace only changes

File: deploy/deploy_common.sh [hunk 4]
```diff
@@ -124,12 +120,22 @@ fi
...
 ANSIBLE_DBG_LOG="/tmp/ansible_debug.log"
 echo "" > $ANSIBLE_DBG_LOG
-echo "Detailed deployment output are available at $ANSIBLE_DBG_LOG and /var/log/ansible.log"
+echo -e "Detailed deployment output are available at $ANSIBLE_DBG_LOG and /var/log/ansible.log\n"
 
 ## compose play options
 PLAY_CMD="ansible-playbook ${ANSIBLE_VERBOSE} \
 -i inventories/hosts_${site} \
 -e \"site=${site} stage=${stage} reg=${reg} DBG_LOG=$ANSIBLE_DBG_LOG\" \
--e \"@inventories/vars/.vault.yml\" \
```
Comment (liangbin-pub) [inline deploy/deploy_common.sh:133]: back it off

File: deploy/deploy_common.sh [hunk 5]
```diff
@@ -124,12 +120,22 @@ fi
...
 -e \"@inventories/vars/site_${site}.yml\" \
 "
+
```
Comment (liangbin-pub) [inline deploy/deploy_common.sh:131]: back it off

File: docs/user/getting_started.md [hunk 6]
```diff
@@ -17,11 +17,11 @@ To demonstrate Zeta functionality End-to-end without the burden of complicated e
 To start developing/deploying Zeta, you will need:
 - a host as **deployer**:
   - to build and package Zeta system
-  - to deploy zeta system to remote site or local as in container based "Kind" scenario
+  - to deploy zeta system to remote site
```
Comment (liangbin-pub) [inline docs/user/getting_started.md:20]: back it off

File: docs/user/getting_started.md [hunk 7]
```diff
@@ -51,7 +51,7 @@ To start developing/deploying Zeta, you will need:
   ```
 - Python modules for remote deployment:
   ```
-  pip3 install boto boto3 ansible
+  pip3 install boto boto3 botocore ansible
```
Comment (liangbin-pub) [inline docs/user/getting_started.md:54]: are you sure?

File: docs/user/getting_started.md [hunk 8]
```diff
@@ -90,8 +90,7 @@ $ kind --version
 $ kubectl version --client
 ```
 
-If you are testing out Zeta with Kind, a script has been included to setup a local Kubernetes cluster, and install all of the components needed to run Zeta.
-Simply run the script below in the Zeta directory.
+If you are testing out Zeta with Kind, a script has been included to setup a local Kubernetes cluster, and install all of the components needed to run Zeta. Simply run the script below in the Zeta directory.
```
Comment (liangbin-pub) [inline docs/user/getting_started.md:94]: Don't make whitespace changes only

File: docs/user/getting_started.md [hunk 9]
```diff
@@ -124,10 +123,12 @@ From the Zeta directory run the following
 ```
 $ ./src/extern/alcor-control-agent/build/build.sh
 ```
-Once ACA is done building, from the Zeta directory, you can now run a simple ping test with the command below.
+Once ACA is done building successfully, the docker image aca_build0 is created. From the zeta directory, you can now run a simple ping test with the command below.
```
Comment (liangbin-pub) [inline docs/user/getting_started.md:126]: back it off, the image name is not needed from user perspective and may be changed later.

**Review by liangbin-pub — CHANGES_REQUESTED** [review]
Please fix review comments

**Comment by kiran1048** [comment]
Creating a new PR.

**Pull request closed without merging**